packages/grafana-data/src/datetime/formats.ts

```diff
@@ -48,6 +47,15 @@ export class SystemDateFormatsState {
       second: '2-digit',
     });
 
+    // ES5 doesn't support `DateTimeFormatOptions.fractionalSecondDigits` so we have to use
+    // a hack with string replacement.
+    this.fullDateMS = this.fullDate.replace('ss', 'ss.SSS');
```

Comment:
Is it possible this is happening even when users *don't* want millisecond precision. For example, I have a table and I have set 

` GF_DATE_FORMATS_FULL_DATE=MMM Do, YYYY HH:mm:ss`.

I see dates of the format `Aug 31st, 2022 17:23:20.067` even though I want `Aug 31st, 2022 17:23:20`

Reply:
Nevermind, it appears my version of grafana does not even have this change.

However, similar logic exists on the older tag which I'm still suspicious of: https://github.com/grafana/grafana/blame/v9.0.7/packages/grafana-data/src/datetime/formats.ts#L36

Reply:
Commented on https://github.com/grafana/grafana/issues/48917.

I think caused in https://github.com/grafana/grafana/pull/37569